cli/src/commands/codegen/index.ts

```diff
@@ -72,6 +75,24 @@ export default class Codegen extends BaseCommand {
       );
     }
 
+    const packageJSON = await readJSON('package.json');
+    if (!packageJSON) {
+      return this.error('Could not find package.json. Please run this command in the root of your project.');
+    }
+
+    const installedVersion: string | undefined = packageJSON.dependencies['@xata.io/client'];
+    if (!installedVersion) {
+      return this.error(
+        `Could not find @xata.io/client in your package.json. Please install it to use the codegen command.`
+      );
+    }
+
+    if (!semver.satisfies(installedVersion, `^${sdkVersion}`)) {
+      return this.error(
+        `The installed version of @xata.io/client (${installedVersion}) does not satisfy the required version (${sdkVersion}). Please update your package.json.`
```

Comment:
```suggestion
        `The installed version of the SDK (${installedVersion}) does not satisfy the CLI version (${sdkVersion}). Please update the packages to run the codegen.`
```

Reply:
I would like to:

- If SDK is outdated, offer to update it with a question
- If CLI is outdated, show command to update (or when we have auto update, offer to update it with a question)

Also:

- Add `--force` to ignore the errors and continue if user knows what they're doing